File: client/src/models/Player.ts
import { Card } from './Cards/Card';

export class Player {
  connectionId!: string;
  hand!: Card[];
  score!: number;
  aerodrome!: boolean;
  airdrop!: boolean;

  constructor() {
    this.hand = [];
    this.score = 0;
    this.aerodrome = false;
    this.airdrop = false;
  }
}

File: server/ALS.Models/Models/Player.cs
namespace ALS.Models.Models;

public class Player
{
    public string ConnectionId { get; set; }
    public string Color { get; set; }   
    public List<Card> Hand { get; set; }
    public int Score { get; set; }
    public string Name { get; set; }
    public bool Aerodrome { get; set; }
    public bool Airdrop { get; set; }
}
